src/main/java/Duke.java

```diff
@@ -1,10 +1,294 @@
+import task.*;
+import storage.Storage;
+import ui.Ui;
+
+import java.util.*;
```

Comment:
Should the * import can be replaced with the individual required imports? I think it was in the coding standard guidelines.

Reply:
Yup! thank you